conf.d/teamcity.yaml.example

```diff
@@ -10,10 +10,14 @@ instances:
     # Specify the server name of your teamcity instance here
     # Guest authentication must be on if you want the check to be able to get data
     server: teamcity.mycompany.com
```

Comment:
Could you mention here that when basic auth is used the username and password need to be provided in the `server` parameter in the `user:password@server` format?